backend/README.md

```diff
@@ -181,7 +177,7 @@ To join the team:
 3. A Badgeyay admin adds you.
 
 To stay a maintainer in the team:
-1. Follow the rules of [CCCC][cccc] or [Badgeyay](.github/CONTRIBUTING.md) and do not violate them willingly or in a harmful way.
+1. Follow the rules of [CCCC][cccc] or [Badgeyay](../.github/CONTRIBUTING.md) and do not violate them willingly or in a harmful way.
```

Comment:
Please remove maintainer section, It is not needed now 👍

Reply:
@ParthS007  Yeah sure!